src/components/ProyectoForm/index.js

```diff
@@ -73,209 +78,250 @@ export default function ProyectoForm({
                 required
-                label="Tipo"
-                id="project_type"
-                name="project_type"
-                value={project.project_type}
-                labelId="tipo"
+                id="name"
+                type="text"
+                label={PROJECT_LABELS.NOMBRE}
+                name="name"
+                value={project.name}
                 onChange={handleChange}
-              >
-                <MenuItem value="staff_augmentation">
-                  Staff Augmentation
-                </MenuItem>
-                <MenuItem value="end_to_end">End to End</MenuItem>
-                <MenuItem value="tercerizado">Tercerizado</MenuItem>
-                <MenuItem value="hibrido">Híbrido</MenuItem>
-              </Select>
-              <FormHelperText>{errors?.project_type?.[0]}</FormHelperText>
-            </FormControl>
-          </Grid>
-          <Grid item xs={12} md={4}>
-            <FormControl fullWidth error={!!errors?.project_state}>
-              <InputLabel id="state-input">Estado *</InputLabel>
-              <Select
+                error={!!errors?.name}
+                helperText={errors?.name?.[0]}
+                inputProps={{ maxLength: 30 }}
+              />
+            </Grid>
+            <Grid item xs={12} md={4}>
+              <FormControl required fullWidth error={!!errors?.project_type}>
+                <InputLabel id="tipo">{PROJECT_LABELS.TIPO}</InputLabel>
+                <Select
+                  fullWidth
+                  required
+                  label={PROJECT_LABELS.TIPO}
+                  id="project_type"
+                  name="project_type"
+                  value={project.project_type}
+                  labelId="tipo"
+                  onChange={handleChange}
+                >
+                  <MenuItem value="staff_augmentation">
+                    Staff Augmentation
+                  </MenuItem>
+                  <MenuItem value="end_to_end">End to End</MenuItem>
+                  <MenuItem value="tercerizado">Tercerizado</MenuItem>
+                  <MenuItem value="hibrido">Híbrido</MenuItem>
+                </Select>
+                <FormHelperText>{errors?.project_type?.[0]}</FormHelperText>
+              </FormControl>
+            </Grid>
+            <Grid item xs={12} md={4}>
+              <FormControl required fullWidth error={!!errors?.project_state}>
+                <InputLabel id="state-input">
+                  {PROJECT_LABELS.ESTADO}
+                </InputLabel>
+                <Select
+                  fullWidth
+                  required
+                  labelId="state-input"
+                  label={PROJECT_LABELS.ESTADO}
+                  id="project_state"
+                  name="project_state"
+                  value={project.project_state}
+                  onChange={handleChange}
+                >
+                  <MenuItem value={PROJECT_LABELS.ESTADO_VERDE_MIN}>
```

Comment:
We could do an array with values and then map and show <MenuItem value={mapValue}></MenuItem>